app/main.py

```diff
@@ -1,3 +1,20 @@
+from math import ceil
+
+
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self, name: str,
+                 description: str,
+                 weeks: int):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days: int):
+        return ceil(days / 7)
+
+    @classmethod
+    def from_dict(cls, course_dict: dict):
+        return OnlineCourse(course_dict["name"],
+                            course_dict["description"],
+                            cls.days_to_weeks(course_dict['days']))
```

Comment:
and here better use `OnlineCourse.days_to_weeks`. `days_to_weeks` is a staticmethod, you are not supposed to call it from child classes